### DISCUSSION TITLE
Question: place cursor at the end of template when exiting

### DISCUSSION THREAD

**Discussion by tbdcit** [minad/tempel#141] [Q&A] [answered]
I think this more likely a question about the tempo syntax, sorry if this not the right place to ask.

What I would like it s a template  which does: `\\( $1 \\)$0`,  where `$1` gets filled with the current selection (if there is one) otherwise places the cursor at `$1` then at the next jump exits the template and places the cursor at `$0`. 

What I have tried is `( mk "\\( "  r " \\)" q)`. This almost gives what I want but template exists immediately after insertion (I cannot jump to q).

**Comment by minad** [comment]
As far as I know this behavior is not supported.

**Reply by tbdcit** [reply]
That’s a shame as it is very useful feature, especially for writing latex. 

Do you think it would be possible to implement such a behaviour?

**Reply by tbdcit** [reply, marked as answer]
A workaround is to use `r` with a prompt. My example above would become `( mk "\\( "  (r "MATHS") " \\)" q)`. 

It would be very nice if this could work without the prompt i.e. to implement a placeholder which behaves like `r` which does not exit. Where would be the correct place to request this feature?
